app/extensions/torrent/index-dev.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="availableLanguages" content="">
+    <meta name="defaultLanguage" content="en-US">
+    <!-- TODO: Don't allow img-src *, needed for favicons -->
+    <!-- TODO: Refactor away all unsafe-inline content -->
+    <!-- TODO: Replace suggestqueries.google.com and ac.duckduckgo.com and other search engines with a single config search engine -->
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; form-action 'none'; referrer no-referrer; script-src 'self' http://localhost:*; connect-src 'self' https://s3.amazonaws.com/adblock-data/ https://s3.amazonaws.com/safe-browsing-data/ https://s3.amazonaws.com/tracking-protection-data/ https://s3.amazonaws.com/https-everywhere-data/ http://localhost:* ws://localhost:* https://suggestqueries.google.com https://ac.duckduckgo.com https://completion.amazon.com https://search.yahoo.com https://api.bing.com https://www.startpage.com https://brave-download.global.ssl.fastly.net https://brave-laptop-updates.global.ssl.fastly.net https://brave-download.global.ssl.fastly.net https://laptop-updates-pre.brave.com https://brave-laptop-updates-pre.brave.com; style-src 'unsafe-inline'; font-src 'self' http://localhost:*; img-src 'self' * data:; object-src 'self'; plugin-types application/browser-plugin">
+    <title>Brave</title>
+    <script src="index-load-script.js" defer></script>
+    <script src="ext/l20n.min.js" async></script>
+    <link rel="localization" href="locales/{locale}/app.properties">
+    <link rel="localization" href="locales/{locale}/common.properties">
+    <link rel="localization" href="locales/{locale}/bravery.properties">
+    <link rel="localization" href="locales/{locale}/menu.properties">
+    <link rel="localization" href="locales/{locale}/downloads.properties">
```

Comment:
yes, and in fact, i think the whole `index-dev.html` should prob be removed. we're just running `npm run webpack` for each change anyway